packages/widgets/src/lib/components/PipelineIcon/PipelineIcon.svelte

```diff
@@ -4,6 +4,7 @@
 	import IconAudioClassification from "../Icons/IconAudioClassification.svelte";
 	import IconAudioToAudio from "../Icons/IconAudioToAudio.svelte";
 	import IconAutomaticSpeechRecognition from "../Icons/IconAutomaticSpeechRecognition.svelte";
+	import IconAudioTextToText from "../Icons/IconAudioToAudio.svelte";
```

Comment:
better if the icon is unique imo, but no big deal. cc @gary149

Reply:
Sure, we also need one for Text to Audio. I'll see if I can make something for them in a follow-up PR.